ui/local-development/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/ubi8/nodejs-16 AS base
+
+RUN npm -g install npm@latest @vue/cli@latest yarn@latest
+
+WORKDIR /app

```

Comment:
The docker build fails for me because the /app dir is not writable.
When I changed it to
```Dockerfile
WORKDIR /opt/app-root/src
```
That's the default workdir for the  registry.access.redhat.com/ubi8/nodejs-16 image

I also added 
```dockerfile
COPY yarn.lock .
```

With those changes `podman-compose -f local-development/docker-compose.yaml up` works but when I change files it does not recompile.